File: app/src/main/java/libertypassage/com/app/models/ModelReset.java

package libertypassage.com.app.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class ModelReset {

    @SerializedName("error")
    @Expose
    private Boolean error;
    @SerializedName("message")
    @Expose
    private String message;
    @SerializedName("bearer_token")
    @Expose
    private String bearer_token;
    @SerializedName("otp")
    @Expose
    private int otp;
    @SerializedName("details")
    @Expose
    private DetailsResetPass details;

    public Boolean getError() {
        return error;
    }

    public void setError(Boolean error) {
        this.error = error;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public String getBearer_token() {
        return bearer_token;
    }

    public void setBearer_token(String bearer_token) {
        this.bearer_token = bearer_token;
    }

    public int getOtp() {
        return otp;
    }

    public void setOtp(int otp) {
        this.otp = otp;
    }

    public DetailsResetPass getDetails() {
        return details;
    }

    public void setDetails(DetailsResetPass details) {
        this.details = details;
    }

}

File: app/src/main/java/libertypassage/com/corporate/model/ModelReset.kt
package libertypassage.com.corporate.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class ModelReset {
    @SerializedName("error")
    @Expose
    var error: Boolean? = null

    @SerializedName("message")
    @Expose
    var message: String? = null

    @SerializedName("bearer_token")
    @Expose
    var bearer_token: String? = null

    @SerializedName("otp")
    @Expose
    var otp = 0

    @SerializedName("details")
    @Expose
    var details: DetailsResetPass? = null
}